meson.build

```diff
@@ -126,12 +126,15 @@ if not get_option('cairo').disabled()
   cairo_dep = dependency('cairo', required: false)
 
   if not cairo_dep.found() and cpp.get_id() == 'msvc'
-    if cpp.has_header('cairo.h')
-      cairo_dep = cpp.find_library('cairo')
-    endif
+    cairo_dep = cpp.find_library('cairo', required: false,
+                                 has_headers: ['cairo.h'])
   endif
 
-  if not cairo_dep.found() and get_option('cairo').enabled()
+  if not cairo_dep.found()
+    # Note that we don't hae harfbuz -> cairo -> freetype2 -> harfbuzz fallback
+    # dependency cycle here because we have configured freetype2 above with
+    # harfbuzz support disabled, so when cairo will lookup freetype2 dependency
+    # it will be forced to use that one.
     cairo_dep = dependency('cairo', fallback: ['cairo', 'libcairo_dep'])
```

Comment:
probably this should have `required: get_option('cairo')`

Reply:
good point, added a commit for that.